src/CoreBundle/private/modules/MTHeader/MTHeader.class.php

```diff
@@ -380,6 +378,8 @@ public function addTabToUrlHistory()
     }
 
     /**
+     * @deprecated since 6.3.0 - not used anymore, we don't show a user icon in the header toolbar anymore.
```

Comment:
Could you add this deprecation to the UPGRADE file?

Reply:
done